### DISCUSSION TITLE
instal.exe Error

### DISCUSSION THREAD

**Discussion by JayBean26** [ppy/osu#17967] [Q&A]
### Type

Other

### Bug description

Upon install I get an error and the download doesn't start

### Screenshots or videos

![unknown](https://user-images.githubusercontent.com/92402968/164994867-32f110e5-8ca5-4920-900b-a5922c8844b7.png)

### Version

2022.424.1

### Logs

not installed

**Comment by frenzibyte** [comment]
Please include details about your operating system.

**Reply by JayBean26** [reply]
Sure
Edition	Windows 10 Home
Version	20H2
Installed on	‎2021-‎11-‎04
OS build	19042.631
Experience	Windows Feature Experience Pack 120.2212.31.0
Device name	DESKTOP-2AP1EAS
Processor	Intel(R) Core(TM) i5-1035G1 CPU @ 1.00GHz   1.19 GHz
Installed RAM	8.00 GB (7.70 GB usable)
Product ID	00325-81676-85441-AAOEM
System type	64-bit operating system, x64-based processor
Pen and touch	Pen support

**Reply by JayBean26** [reply]
I have installed the game before though

**Reply by frenzibyte** [reply]
I see, hard to say with just that error code which is just another form of -1. https://github.com/Squirrel/Squirrel.Windows/issues/1734 is about the closest thread I can find that *may* be of relevance here, cc @caesay.

**Reply by JayBean26** [reply]
I'll give that a shot?

**Reply by caesay** [reply]
@frenzibyte The linked issue is probably not relevant - the issue there was caused by publishing invalid updates (user error) and not a problem with Squirrel. 

For this issue, please check to see if a log file exists here, and share it if so: `%localappdata%\SquirrelClowdTemp\Squirrel.log` ?
